### PR TITLE
Update activation logic for default username information

### PR DESCRIPTION
**Pull request by AnanyaJha**
### What does this PR do?
This PR updates the activation logic for the org picker, org browser, and workspace org type to only collect default username information once.

### CODE REVIEW

File: packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts [hunk 1]
```diff
@@ -35,14 +35,8 @@ export class OrgList implements vscode.Disposable {
...
 
-  public async displayDefaultUsername() {
-    let defaultUsernameorAlias: string | undefined;
-    if (hasRootWorkspace()) {
-      defaultUsernameorAlias = await OrgAuthInfo.getDefaultUsernameOrAlias(
-        false
-      );
-    }
-    if (defaultUsernameorAlias) {
+  public async displayDefaultUsername(defaultUsernameorAlias?: string) {
```
Comment (lcampos) [inline packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts]: This doesn't need to be `async` anymore.

File: packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts [hunk 2]
```diff
@@ -177,8 +171,14 @@ export class OrgList implements vscode.Disposable {
...
-    await setupWorkspaceOrgType();
-    await this.displayDefaultUsername();
+    let defaultUsernameorAlias: string | undefined;
+    if (hasRootWorkspace()) {
+      defaultUsernameorAlias = await OrgAuthInfo.getDefaultUsernameOrAlias(
+        false
+      );
+    }
+    await setupWorkspaceOrgType(defaultUsernameorAlias);
+    await this.displayDefaultUsername(defaultUsernameorAlias);
```
Comment (lcampos) [inline packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts]: You can remove this `await`.

File: packages/salesforcedx-vscode-core/src/index.ts [hunk 3]
```diff
@@ -398,16 +400,21 @@ export async function activate(context: vscode.ExtensionContext) {
...
 
+  let defaultUsernameorAlias: string | undefined;
+  if (hasRootWorkspace()) {
+    defaultUsernameorAlias = await OrgAuthInfo.getDefaultUsernameOrAlias(false);
+  }
+
   // register org picker commands and set up filewatcher for defaultusername
   const orgList = new OrgList();
-  await orgList.displayDefaultUsername();
+  await orgList.displayDefaultUsername(defaultUsernameorAlias);
```
Comment (lcampos) [inline packages/salesforcedx-vscode-core/src/index.ts]: You can remove this `await` after removing the `async` from `displayDefaultUsername`

File: packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts (hunk 1 above)
Comment (AnanyaJha) [reply packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts]: Ah good catch

File: packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts (hunk 2 above)
Comment (AnanyaJha) [reply packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts]: Cool

File: packages/salesforcedx-vscode-core/src/index.ts [hunk 4]
```diff
@@ -352,10 +352,12 @@ function registerOrgPickerCommands(orgList: OrgList): vscode.Disposable {
 }
 
 async function setupOrgBrowser(
```
Comment (brpowell) [inline packages/salesforcedx-vscode-core/src/index.ts]: Can this setup function be defined in the orgBrowser module and imported instead? Might be good to reduce crowding this file when we can.

Comment (AnanyaJha) [reply packages/salesforcedx-vscode-core/src/index.ts]: I was thinking about that initially too but ended up deciding against it for now. If we want to move these helper setup methods over to their respective files, I can definitely do that in another pr.

**Pull request merged**